modules/osd-create-cluster-ccs.adoc

```diff
@@ -279,6 +279,13 @@ ifdef::osd-on-gcp[]
 If you are installing a cluster into a Shared VPC, the VPC name and subnets are shared from the host project.
 ====
 endif::osd-on-gcp[]
+ifdef::osd-on-aws[]
+.. Optional: Expand *Additional security groups* and select additional custom security groups to apply to nodes in the machine pools created by default. You must have already created the security groups and associated them with the VPC you selected for this cluster. You cannot add or edit security groups to the default machine pools after you create the cluster.
++
+By default, the security groups you specify will be added for all node types. Uncheck the *Apply the same security groups to all node types* checkbox to apply different security groups for each node type.
++
+For more information, see the requirements for _Security groups_ under _Additional resources_.
```

Comment:
```suggestion
For more information, see the requirements for security groups in the "Additional resources" section.
```
- See IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#highlighting-by-element
- No highlighting is needed for the phrase "security groups" in this sense. If you were referring to a title in the documentation, you could use double-quotation marks.
- I updated the highlighting of "Additional resources" from italics to double-quotation marks per the IBM Style "Highlighting" topic.